README.adoc

```diff
@@ -301,62 +304,67 @@ and trigger a failure if a change introduces a defect.
 
 === Setting up your tests
 
-EndpointTest.java
+EndpointIT.java
 [source, java, linenums, role='code_column hide_tags=comment']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/hateoas/EndpointTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/hateoas/EndpointIT.java[]
 ----
 
 [role="code_command hotspot" subs="quotes"]
 ----
-#Create the `EndpointTest` class.#
-`src/test/java/it/io/openliberty/guides/hateoas/EndpointTest.java`
+#Create the `EndpointIT` class.#
+`src/test/java/it/io/openliberty/guides/hateoas/EndpointIT.java`
 ----
 
-You can use the [hotspot=Before]`@Before` and [hotspot=After]`@After` annotations to perform any setup and teardown tasks for each
+You can use the [hotspot=Before]`@BeforeEach` and [hotspot=After]`@AfterEach` annotations to perform any setup and teardown tasks for each
 of your individual tests.
 
 === Writing the tests
 
-Each test method must be marked with the [hotspot=Test]`@Test` annotation. Typically, the execution order of test
-methods is not controlled, but if control is required, you can place methods in a single container test
-method. This container method is then the only one marked with the [hotspot=Test]`@Test` annotation. The following
-[hotspot=testSuite]`testSuite()` contains two test methods, which run in the order they appear:
-
-The test [hotspot=testLinkForInventoryContents]`testLinkForInventoryContents` is responsible for asserting that
-the correct HATEOAS link is created for the inventory contents.
+Each test method must be marked with the [hotspot=Test1 hotspot=Test2]`@Test` annotation. The execution order of test methods
+is controlled by marking them with the [hotspot=Order1 hotspot=Order2]`@Order` annotation. The value passed into the annotation
+denotes the order in which the methods are run.
 
+The test class contains three helper methods and two test methods:
 
 The [hotspot=getResponse]`getResponse` helper method reuses the same line of code for retrieving a response from a
 specific URL. This technique helps keep your code neat and organized.
 
-The [hotspot=assertResponse]`assertResponse` method ensures that the response code that you receive is
+The [hotspot=assertResponse]`assertResponse` helper method ensures that the response code that you receive is
 valid (200).
 
-The [hotspot=testLinksForSystem]`testLinksForSystem` test is responsible for asserting that the correct
+The method [hotspot=visitLocalhost]`visitLocalhost` creates a [hotspot=targetResponse]`GET` request to the
+`system` service, registering the `localhost` system.
+
+The test [hotspot=testLinkForInventoryContents]`testLinkForInventoryContents` is responsible for asserting that
+the correct HATEOAS link is created for the inventory contents.
+
+Finally, the test [hotspot=testLinksForSystem]`testLinksForSystem` test is responsible for asserting that the correct
```

Comment:
Change
"Finally, the test `testLinksForSystem` test is responsible for asserting that the correct HATEOAS links are created for the `localhost` system. This method checks for both the `self` link that points to the `inventory` service and the `properties` link that points to the `system` service that is running on the `localhost` system."
To
"Finally, the `testLinksForSystem` test is responsible for asserting that the correct HATEOAS links are created for the `localhost` system. This method checks for both the `self` link that points to the `inventory` service and the `properties` link that points to the `system` service, which is running on the `localhost` system."